lib/client/Client.js

```diff
@@ -664,6 +665,30 @@ export default class Client {
     return Date.now();
   }
 
+  /**
+   * Sets the singleton {@link FilterRule} instance for a given account.<br />
+   * This will send a `setFilter` request to the JMAP backend.
+   *
+   * @param filteringRules {FilterRule[]} The {@link FilterRule} instance to set
```

Comment:
It's very ambiguous, the param name is a plural, the type is an array and the description is not plural anymore